src/lib/forms-validation/common.ts

```diff
@@ -14,13 +15,24 @@ export function validateJoiSchema<T>(
   return result.value as T
 }
 
-export const getRootFormElements = (
+/**
+ * Reduce an array of form elements down to the elements that are not purely to
+ * determine the layout of the form. E.g. "page" based elements.
+ *
+ * @param elements
+ * @returns
+ */
+export const stripLayoutFormElements = (
```

Comment:
Neither function name fully captures what the function does, but to me getRootFormElements at least tells me I'm not getting sub-forms and repeatable set elements etc

Reply:
> Neither function name fully captures what the function does
agreed but with the docs on the function, I hope its clear now.

Reply:
somewhat, could be clearer.  Ie, stating that it won't dive into repeatable sets.

Reply:
repeatable sets are not layout elements.

> the elements that are not purely to determine the layout of the form

surely all the devs know that repeatable sets are a lot more than just form layout.